modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/loader/parser/java/JavaExtensionModelParserTestCase.java

```diff
@@ -44,6 +47,15 @@ public class JavaExtensionModelParserTestCase {
   @Rule
   public ExpectedException expectedException = ExpectedException.none();
 
+  @Test
+  @Description("Verify that an extension with Jdk field with encapsulated internal fields does not fail due to reflection in Java 17")
+  public void getParameterizedWithJavaFieldsExtensionUsingSdkApi() {
+    List<MetadataType> importedTypes = getParser(ParameterizedWithJavaTypeExtension.class).getImportedTypes();
+    assertThat(importedTypes.size(), is(1));
+    assertThat(importedTypes.get(0).getAnnotation(ClassInformationAnnotation.class).get().getClassname(),
```

Comment:
use a proper matcher for the condition you are checking instead of `is(true)`

Reply:
Removed that check as per earlier comment by Mariano